src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,33 @@
+@import "./utils/variables.scss";
+
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  gap: $card-margin-y $card-margin-x;
+  max-width: min-content;
+  margin: 0 auto;
```

Comment:
Please avoid adding external styles such as margins or positioning to BEM blocks.